_sass/blocks/case-studies/_maps.scss

```diff
@@ -0,0 +1,19 @@
+.case_studies-map {
+	stroke: $mid-gray;
```

Comment:
Line should be indented with spaces, not tabs<br>Properties should be ordered fill, stroke, stroke-width